MAUI/ImageEditor/undo-redo.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Undo Redo in .NET MAUI Image Editor Control | Syncfusion
+description: Learn about Undo Redo support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Undo Redo in .NET MAUI Image Editor (SfImageEditor)
+
+One of the important features of the image editor control is to perform `Undo` and `Redo` operations for adding shapes, text, effects, and drawing paths. 
```

Comment:
One of the important features of the image editor control is to perform `Undo` and `Redo` to revert the changes.

Reply:
Modified a content